inst/rmarkdown/templates/thesis/skeleton/skeleton.Rmd

```diff
@@ -41,10 +41,16 @@ mini-lof: false  # mini-list of figures by start of each chapter?
 ####################
 ## Bibliography   ##
 ####################
-bibliography: bib/references.bib
+bibliography: bib/references.bib #Set your bibliography file here.
+packagebibliography: bib/packages.bib #Sets the automatically generated R-Package Bibliography.
```

Comment:
If this is done automatically, does it have to be listed here in the skeleton?

Reply:
True, I can define it directly in the tex template but if the user wants to change the file path of the bibliography, this would give him/her a greater level of flexibility.